com.github.tmewett.BrogueCE.json

```diff
@@ -0,0 +1,53 @@
+{
+    "app-id": "com.github.tmewett.BrogueCE",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "brogue",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--share=network"
+    ],
+    "modules": [
+        {
+            "name": "brogue",
+            "buildsystem": "simple",
+            "build-commands": [
+                "export DATADIR=/app/lib/brogue/",
+                "make DATADIR=/app/lib/brogue/",
+                "install -Dm755 bin/brogue /app/lib/brogue/brogue",
+                "install -Dm755 brogue /app/bin/brogue",
+                "cp -r bin/assets /app/lib/brogue/assets",
+                "install -Dm644 bin/assets/icon.png /app/share/icons/hicolor/256x256/apps/com.github.tmewett.BrogueCE.png",
+                "install -Dm644 brogue.desktop /app/share/applications/com.github.tmewett.BrogueCE.desktop"
```

Comment:
The metainfo file should be installed here

Reply:
Ah yes, thanks i have fixed that also, fingers crossed now ha!